src/Testcontainers/Clients/ContainerConfigurationConverter.cs

```diff
@@ -8,15 +8,15 @@ namespace DotNet.Testcontainers.Clients
   using DotNet.Testcontainers.Networks;
   using JetBrains.Annotations;
 
-  internal sealed class TestcontainersConfigurationConverter
+  internal sealed class ContainerConfigurationConverter
```

Comment:
What is a `ConfigurationConverter`?

Reply:
It maps (converts) TC for .NET types to Docker.DotNet types. There is not always a 1:1 mapping.